numpy/distutils/system_info.py

```diff
@@ -1698,14 +1698,14 @@ def calc_info(self):
             lib = self.has_cblas(info)
             if lib is not None:
                 info['language'] = 'c'
-                info['libraries'] = [lib]
+                info['libraries'] = lib
                 info['define_macros'] = [('HAVE_CBLAS', None)]
         self.set_info(**info)
 
     def has_cblas(self, info):
+        """ changed in 1.17 to return a list instead of a str """
         # primitive cblas check by looking for the header and trying to link
         # cblas or blas
-        res = False
```

Comment:
The comments should be turned into a proper docstring, "changed in ..." should be the last line. This might be a good place to describe the search order priority, the new comment below can be part of the docstring.

Reply:
Added, remarked in the `Returns` segment when the list appeared. I have refrained from describing what it was as I think that is superfluous information.